nuts/Issues/agb/description.xml

```diff
@@ -53,8 +52,8 @@
 				<nl>AGB-code</nl>
 			</Name>
 			<Description>
-				<en>Uniquely identifying key for individual care providers and care organizations</en>
-				<nl>Uniek identificerende sleutel van individuele zorgverleners en zorgorganisaties</nl>
+				<en>Uniquely identifying code for individual healtcare professional</en>
```

Comment:
`healthcare`

Reply:
Nicely spotted @confiks ! @bajacobs I will fix this once we have agreed on the other issues...

Reply:
Right.